Pods/CatchingFire/src/CatchingFire.swift

```diff
@@ -8,6 +8,10 @@
 
 import XCTest
 
+#if !swift(>=3.0)
```

Comment:
Ah, I just noticed this change is for the dependency `CatchingFire`. The changes would need to be made there and released to be added here. https://github.com/mrackwitz/CatchingFire